app/controllers/users_controller.rb

```diff
@@ -133,8 +133,10 @@ def create
     @user = User.new(user_params)
     @user.reset_validation!
 
+    captcha_message = 'Sorry, but the reCAPTCHA was incorrect.'
```

Comment:
Is this the error message the user sees? Hmmm.. if yes, maybe we could change it to say something like, "The text you entered in does not match the text in the image." It's possible our users don't know what recaptcha is.